addons/escoria-core/game/core-scripts/esc/commands/hide_menu.gd

```diff
@@ -1,10 +1,14 @@
 # `hide_menu menu_type [enable_automatic_transition]`
 #
-# Hides either the main menu or the pause menu.
+# Hides either the main menu or the pause menu. The enable_automatic_transition
+# parameter can be used to specify if Escoria manages the graphical transition
+# or not. If set to false, you can perform your own transition as part of the
+# room's `setup` event (if you want to change the look of the transition used
+# from the default for example) using the `transition` command.
```

Comment:
@StraToN - Comment from @BHSDuncan on my earlier PR : "I actually think you do it as part of both when exiting a room and when entering a room, so not just setup but also the exiting event in the previous room. @StraToN those correspond to transitioning out and in, no?"

Reply:
Yes, that's exactly it. 
If you disable automatic transitions, then you have to manage the transition out as you exit the scene (so in the :exit event, for an ESCExit) *and* you also have to manage the transition in the new room (at the end of the :setup event of the new room, usually)

Reply:
@balloonpopper Does this answer your question? May I mark the discussion as resolved?

Reply:
@StraToN - I'll need to update the text as I only mentioned transition in. 
If I'm reading the code correctly, it appears the ":exit" event is actually ":exit_scene". Can you please confirm and then I'll update the documentation to add
"Transitions when leaving a room can be managed manually as part of the `exit_scene` event."

Reply:
Correct, the actual event name is `exit_scene`, not `exit`.

Reply:
I'll leave updating the documentation for hide_menu for now. I want to clarify when you'd expect to use it (see [issue 179](https://github.com/godot-escoria/escoria-issues/issues/179) ) and will modify the text accordingly.